src/main/java/org/opentripplanner/middleware/bugsnag/BugsnagDispatcher.java

```diff
@@ -20,15 +20,15 @@
 import static org.opentripplanner.middleware.spark.Main.getConfigPropertyAsText;
 
 /**
- * Responsible for getting required information from Bugsnag. This is done by making calls to Bugsnag's API endpoints
+ * Responsible for getting {@link BugsnagEvent} information from Bugsnag. This is done by making calls to Bugsnag's API endpoints
  * with a valid authorization token (BUGSNAG_API_KEY).
- *
+ * <p>
  * A bugsnag API key is a key that is unique to an individual Bugsnag user. This key can be obtained by logging into
  * Bugsnag (https://app.bugsnag.com), clicking on settings (top right hand corner) -> “My account settings”. From here
  * select “Personal auth tokens” and then “Generate new token”.
- *
+ * <p>
  * The following Bugsnag API endpoints are currently used:
- *
+ * <p>
```

Comment:
If using HTML inside JavaDoc, this could be `<ul>...</ul>` with each bullet being an `<li>...</li>`

Reply:
Removed HTML markup.